<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:24pt; font-weight:600; color:#204a87;&quot;&gt;tSCS stimulation&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_quit">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>300</y>
      <width>89</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>QUIT</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>73</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_P1">
       <property name="text">
        <string>Protocol 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_P2">
       <property name="text">
        <string>Protocol 2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_P3">
       <property name="text">
        <string>Protocol 3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_P4">
       <property name="text">
        <string>Protocol 4</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>90</y>
      <width>161</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="Button_10Pulses">
       <property name="text">
        <string>10 SINGLE STIMULI</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Button_2CloseStim">
       <property name="text">
        <string>2 CLOSE STIMULI</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Button_Ramp">
       <property name="text">
        <string>RAMP</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Button_ContinuousStim">
       <property name="text">
        <string>CONTINUOUS 
 STIMULATION</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>535</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHOME">
    <property name="title">
     <string>HOME</string>
    </property>
   </widget>
   <addaction name="menuHOME"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
